templates/init/functions/typescript/_eslintrc

```diff
@@ -10,6 +10,8 @@ module.exports = {
     "plugin:import/warnings",
     "plugin:import/typescript",
     "google",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
If we were to add these, I would rather put them immediately below the `eslint:recommended` line so that the following entries can override as appropriate.

Reply:
Yes this makes sense. I think `"plugin:@typescript-eslint/eslint-recommended"` it's not really needed. I'll change that